helm/efk-stack-app/charts/opendistro-certs/values.yaml

```diff
@@ -4,8 +4,10 @@
 
 enabled: true
 
-image: quay.io/giantswarm/opendistro-certs
-imageTag: v0.0.3
+image:
+  registry: quay.io
```

Comment:
```suggestion
```
I think these are not needed, since global ones are being used. Even for doc purposes, we document just the one in parent chart values.yaml. Happa likely doesn't display anything more anyway.

Reply:
just to bring the conversation here instead of slack - I was trying to avoid changing the chart as much as possible. I have some things I'd like to discuss with the efk app squad when they're back - this is one of them.

Reply:
I think for now this should be ok, but the imho the use of globals makes sense and maybe we should submit a PR upstream to use it.

Reply:
@yasn77 the upstream charts are all separate entities - the root values file is of our own making for the purpose of binding all these different charts together.